sdk/log/internal/x/x.go

```diff
@@ -0,0 +1,46 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+// Package x contains support for OTel log SDK experimental features.
+package x // import "go.opentelemetry.io/otel/sdk/log/internal/x"
```

Comment:
nit note (just for knowledge sharing)

Recently I learned that in Go team's codebase [`x`](https://pkg.go.dev/golang.org/x) stands for "external"  and [`exp`](https://pkg.go.dev/golang.org/x/exp) is for "experimental".

Nothing to change thought 😉 

PS. I think it was me who proposed the `x` name so I just wanted to confess.

Reply:
I think we can leave this as is for this PR so we are consistent with `sdk/metric/internal/x`. I'm happy to approve a change afterwards that moves this and the metric `x` package to new names.